Setting up GraphQL

Sam09040 opened the pull request:
Setting GraphQL up using a basic schema, also using express server and GraphiQL.

Sam09040 committed:
Setting up GraphQL

Setting GraphQL up using a basic schema, also using express server and GraphiQL.

walteraandrade commented:
@Sam09040 você fez um upload com o `node_modules`, essa pasta não deve vir pro github. 
Pra evitar isso, usamos um arquivo chamado `gitignore`, você pode parender mais sobre ele [aqui](https://git-scm.com/docs/gitignore)

Sam09040 committed:
Adding .gitignore

Adding .gitignore file and deleting all of the files from the node_modules directory.

racsandoval commented on server.js:
```diff
@@ -0,0 +1,51 @@
+var app = express();
+
+// Runs the GraphQL query and prints the response
+graphql({
+    schema,
+    source: "{hello, name}",	
+    rootValue
+}).then(response => {
+    console.log(response)
+});
```
aqui ele está executando uma chamada pro graphql, faz sentido manter esse código aqui?

Sam09040 replied:
realmente, não faz mais, já que dá para ver isso pelo graphiql. eu tava usando essa parte para verificar aquele erro que eu estava tendo, mas não me atentei a apagar.

racsandoval commented on package.json:
```diff
@@ -0,0 +1,30 @@
+  },
+  "homepage": "https://github.com/feature/setup-graphql#readme",
+  "dependencies": {
+    "express": "^4.21.0",
```
geralmente fixamos a versão das dependências para evitar que o código quebre sozinho com mudanças nas libs
```suggestion
    "express": "4.21.0",
```

Sam09040 committed:
Small changes to main and package

Fixing dependencies versions and removing graphql call to console.

Sam09040 committed:
Adding prettier and eslint

Adding prettier and eslint modules for code formatting and standardization.
